client/components/promo-section/promo-card/cta.tsx

```diff
@@ -79,6 +85,33 @@ const PromoCardCta: FunctionComponent< Props > = ( { cta, learnMoreLink, isPrima
 			  };
 	}
 
+	/*
+	 * Shows a notice for whether a promoted feature is
+	 * included in the user's current plan. For layout
+	 * reasons, we show this or learn more link, but not both.
+	 *
+	 * If featureIncludedInPlan is not passed (undefined), then
+	 * nothing will show. If featureIncludedInPlan is true, we show
```

Comment:
I'm generally not fan of "magic" that happens in component depending on what props are passed, but I understood it doesn't make sense to show both links either?

Since this is pretty important aspect how the component works, making docs more visible would be important. https://github.com/Automattic/wp-calypso/pull/81915/files#r1330650893

Reply:
> not fan of "magic" that happens in component depending

In this case, we're just conditionally rendering elements depending on props. That doesn't seem too magic-y :). But I understand (as with comment by @sirbrillig above) that it may not be obvious that display of the Included-in-Plan notice effectively depends on two props (learnMoreLink and featureIncludedInPlan). 

If we want the option to show all three elements (CTA button, Learn More Link, and Included-in-Plan notice), then we will probably need to discuss design. That creates the possibility of three columns in the card / action footer area. Given the limited width of the cards, one of those will need to wrap to the next line. I can just implement that and make a judgement call on design / spacing / layout - and I considered doing that originally. But my guess is that it will trigger a broader design conversation.

Reply:
Yeah exactly, so I also think very clear docs will get us to a good enough place. 👍

Reply:
Given two comments about this same behavior, I may play a bit with allowing all three items to be rendered and see if there's a graceful way to handle that. If so, folks can always choose not to render all three. If it seems too awkward, I'll add the prop doc as suggested and move on for now.

Reply:
@simison @sirbrillig - I can remove the extra logic where the availability notice also depends on the learnMoreLink. If so, it creates the possibility of three items/columns in the action area. It's not the end of the world, but produces maybe-less-than-ideal-styling like this. Let me know if you'd prefer to allow this, or pursue an approach where we only do one or the other, or have other suggestions. 

<img width="1223" alt="availability" src="https://github.com/Automattic/wp-calypso/assets/21228350/d869bcb1-71e2-4f5e-adb8-461588f11bc5">

Reply:
I'd say that the two-line version doesn't look as nice but I don't have a great suggestion for a different way to do it. I'm personally fine with only allowing one or the other as long as it's clear in the docs for each option what the priority is. (If you wanted to be really clever, it could be one prop that has multiple possible values; something like `secondaryContent: CtaAction | 'included'`)

Reply:
Thanks! I think that to keep this moving, I'll just update the docs for the prop for now, and have it display one or the other. I'd be fine updating this in follow up if we decide it's worth it. Just pushed the updated prop documentation.

Reply:
Looks great to me!

Reply:
👍 🚢